src/test/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderChargeTransformerTest.kt

```diff
@@ -68,4 +70,57 @@ class OffenderChargeTransformerTest {
     assertThat(offenderHistoryDetails.secondaryResultDescription).isEqualTo(offenseResult2.description)
     assertThat(offenderHistoryDetails.statuteCode).isEqualTo(statuteCode)
   }
+
+  @Test
+  fun `should set sentence information to null if no associated sentences`() {
+    val offenderHistoryDetails = OffenderCharge.builder()
+      .offenderBooking(OffenderBooking.builder().bookingId(2L).build())
+      .offenderCourtCase(OffenderCourtCase.builder().id(1L).build())
+      .offence(Offence.builder().code("code").description("desc").statute(Statute.builder().code("st code").build()).build())
+      .build().let {
+        OffenderChargeTransformer().convert(it)
+      }
+    assertThat(offenderHistoryDetails.sentenceStartDate).isNull()
+    assertThat(offenderHistoryDetails.primarySentence as Boolean?).isNull()
```

Comment:
had to use `as Boolean?` here to stop a null pointer as it tried to convert from the java to kotlin `Boolean`

Reply:
Nice; I wondered what the fix for that was